docs/concepts/upgrade-support.md

```diff
@@ -63,8 +63,8 @@ where setting the `upgradeConstraintPolicy` to:
 
 OLM supports Semver to provide a simplified way for package authors to define compatible upgrades. According to the Semver standard, releases within a major version (e.g. `>=1.0.0 <2.0.0`) must be compatible. As a result, package authors can publish a new package version following the Semver specification, and OLM assumes compatibility. Package authors do not have to explicitly define upgrade edges in the catalog.
 
-> [!NOTE]
-> Currently, OLM 1.0 does not support automatic upgrades to the next major version. You must manually verify and perform major version upgrades. For more information about major version upgrades, see [Manually verified upgrades and downgrades](#manually-verified-upgrades-and-downgrades).
+!!! note
+    Currently, OLM 1.0 does not support automatic upgrades to the next major version. You must manually verify and perform major version upgrades. For more information about major version upgrades, see [Manually verified upgrades and downgrades](#manually-verified-upgrades-and-downgrades).
```

Comment:
Could we please have a PR or commit for "Fix markers to render in the docs properly"?

Reply:
Here the goal would be change all places where has [!NOTE] for !!! note but it is missing a place: https://github.com/search?q=repo%3Aoperator-framework%2Foperator-controller%20%3E%20%5B!NOTE%5D&type=code

Reply:
@camilamacedo86 I would really prefer not to do that. I'm going page by page through the docs in order. Make changes and verify them (e.g. rendering pages, running examples against the cluster, etc). Splitting into separate things adds a lot of overhead into tedious work.

Splitting this would require me to spend enormous amount of time doing all this again.

Reply:
Also - we don't want to replace all the GitHub markdown flavour in the repo. We only do that in `docs` directory (docs rendered on the website). Everything else is fine as it is because most likely it will be rendered by GitHub.

Reply:
That's a good question @camilamacedo86 . It's not immediately obvious, but only the documentation inside the /docs directory is rendered into [mkdocs](https://squidfunk.github.io/mkdocs-material/reference/). Any markdown files outside of that directory are for rendering on Github. So we wouldn't want to change [hack/tools/catalogs/README.md](https://github.com/operator-framework/operator-controller/blob/aaa0e0090ec0a04bbc990bbda0a4146852cf70bf/hack/tools/catalogs/README.md?plain=1#L30).

Reply:
Thank you a lot for the clarification !!!

Reply:
Moved to https://github.com/operator-framework/operator-controller/pull/1446

Resolving